doc/doxygen/headers/c++11.h

```diff
@@ -17,64 +17,14 @@
 /**
  * @defgroup CPP11 deal.II and the C++11 standard
  *
- * At present, deal.II only requires a compiler that conforms to the
- * <a href="http://en.wikipedia.org/wiki/C%2B%2B#Standardization">C++98</a>
- * standard and does not rely on compilers to either
- * provide the features introduced in
- * <a href="http://en.wikipedia.org/wiki/C%2B%2B03">C++03</a> or
- * <a href="http://en.wikipedia.org/wiki/C%2B%2B11">C++11</a>
+ * Since version 9.0, deal.II requires a compiler that supports at
+ * least <a href="http://en.wikipedia.org/wiki/C%2B%2B11">C++11</a>.
+ * As part of this, many places in the internal implementation of
+ * deal.II is now using features that were only introduced in C++11.
```

Comment:
should be `many places in the internal implementation of deal.II are now using`

Reply:
You came too late, but I fixed it separately -- see #5091.